docs/index.md

```diff
@@ -109,24 +111,24 @@ POST {json file} <url>/insert/
 | version          | Text         | "0.2.1"                                                                         | Follows Semantic Versioning 2.0.0 http://semver.org/spec/v2.0.0.html                             |
 | timestamp        | TimeDate     | yyyy-mm-ddThh:mm:ss.sss-hh:mm                                                   | Current Timestamp. RFC 3339 (ISO 8601 derivative).                                               |
```

Comment:
I made a number of changes based on your comments, and to make the documentation more consistent. I moved several "constraints" to "notes", because they were really notes. I also added `<N/A>` whenever there were no constraints to be clear that N/A doesn't mean "use an N or an A". 

Most of these changes are pretty clear, but there was one thing that I noticed for the first time that might be important; the visit array had a health center id which was not in the documentation.  I don't think we need that, I don't think we can expect that (or I know how to document it), so I removed it.  

I also reordered the array fields to match the documentation order to make it more clear / easier to check.